app/routes/__index/index.tsx

```diff
@@ -21,32 +21,38 @@ export default function IndexRoute() {
         paddingY={{ base: '2', md: '7' }}
         gap={{ md: '16' }}
       >
-        <Flex width={{ base: '100%', md: '50%' }} textAlign="center" padding="5">
-          <Text fontSize={{ base: 'sm', md: 'lg' }}>
-            Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum
-            has been the industry's standard dummy text ever since the 1500s, when an unknown
-            printer took a galley of type and scrambled it to make a type specimen book.
+        <Flex
+          width={{ base: '100%', md: '75%' }}
+          textAlign="center"
+          padding="5"
+          flexDirection="column"
+        >
+          <Heading size="lg">Welcome to My.Custom.Domain!</Heading>
+          <Text fontSize={{ base: 'sm', md: 'lg' }} mt={4}>
+            My Custom Domain creates a unique web address for your project that makes it easier to
+            access on the internet. My Custom Domain also generates a HTTPS certificate to provide a
+            secure connection between client and your domain.
```

Comment:
```suggestion
            My Custom Domain allows you to create unique web addresses for your projects so that they can be
            be accessed on the internet. You can also generate a HTTPS certificate to provide a
            secure connection between client and your domain.
```
Maybe we should say DNS Records (or domains/subdomains?) instead of web addresses, but I am not sure